app/Models/User.php

```diff
@@ -4,12 +4,14 @@
 
 // use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\HasMany;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
+use Spatie\Permission\Traits\HasRoles;
 
 class User extends Authenticatable
 {
-    use HasFactory, Notifiable;
+    use HasFactory, Notifiable, HasRoles;
```

Comment:
At the moment, there's nothing stopping someone setting up a bot and creating 1000's of fake accounts.

We should add email verification to this application, which with Laravel, is pretty straight forward!

https://laravel.com/docs/11.x/verification

You'll notice that in the `web.php` file, all of the routes are set up to handle this already, using Middleware:

```php
->middleware(['auth', 'verified'])
```